common/src/main/kotlin/io/homeassistant/companion/android/common/data/prefs/PrefsRepository.kt

```diff
@@ -1,17 +1,47 @@
 package io.homeassistant.companion.android.common.data.prefs
 
+import androidx.appcompat.app.AppCompatDelegate
 import io.homeassistant.companion.android.common.data.integration.ControlsAuthRequiredSetting
 import io.homeassistant.companion.android.common.util.GestureAction
 import io.homeassistant.companion.android.common.util.HAGesture
 
+enum class NightModeTheme(val storageValue: String) {
+    LIGHT("light"),
+    DARK("dark"),
+
+    // TODO double check this
+    @Deprecated("Kept for backwards compatibility")
+    ANDROID("android"),
```

Comment:
I'm not sure about this one.

Reply:
That's correct. There used to be different settings for `android` and `system` but they did the same, see my PR where I merged them #2923.